apollo-configservice/src/test/java/com/ctrip/framework/apollo/configservice/integration/ConfigControllerIntegrationTest.java

```diff
@@ -128,7 +128,7 @@ public void testQueryGrayConfigWithDefaultClusterAndDefaultNamespaceOK() throws
   @Sql(scripts = "/integration-test/cleanup.sql", executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
   public void testQueryGrayConfigWithDefaultClusterAndDefaultNamespaceAndIncorrectCase() throws Exception {
     AtomicBoolean stop = new AtomicBoolean();
-    periodicSendMessage(executorService, assembleKey(someAppId, ConfigConsts.CLUSTER_NAME_DEFAULT, ConfigConsts.NAMESPACE_APPLICATION),
+    periodicSendMessage(executorService, generate(someAppId, ConfigConsts.CLUSTER_NAME_DEFAULT, ConfigConsts.NAMESPACE_APPLICATION),
```

Comment:
Typically, tests should not rely on the code's logic, as this could overlook certain errors. For instance, if the `generate` function were to be accidentally altered, identifying the issue would be possible by maintaining an independent logic within the `assembleKey`.

Reply:
You're right, let me adjust it.